packages/engine/Source/Scene/GltfLoader.js

```diff
@@ -2453,6 +2473,20 @@ async function loadStructuralMetadata(
   return structuralMetadataLoader.load();
 }
 
+async function loadMeshPrimitiveGpm(loader, gltf, extension, frameState) {
+  const meshPrimitiveGpmLoader = new GltfMeshPrimitiveGpmLoader({
+    gltf: gltf,
+    extension: extension,
+    gltfResource: loader._gltfResource,
```

Comment:
I'm all a fan of a _proper_ concept of `private` (and immutability and all other aspects of clean code). But the pattern of accessing `_underscore` variables (sometimes even `in._some._chains`) is so abundant that it can likely not be addressed here sensibly.